notifications_utils/polygons.py

```diff
@@ -217,7 +219,10 @@ def as_coordinate_pairs_long_lat(self):
         order, for example leaflet.js.
         '''
         return [
-            [[x, y] for x, y in polygon.exterior.coords]
+            [[
+                round(x, self.output_precision_in_decimal_places),
+                round(y, self.output_precision_in_decimal_places),
+            ] for x, y in polygon.exterior.coords]
             for polygon in self
         ]
 
```

Comment:
Do you also need to add the precision to the next method `as_coordinate_pairs_lat_long`?

Reply:
That method re-uses the output from this one, so it will get the same precision automatically.

Reply:
So it does - soz